Header header

float64 clutch
float64 brake
float64 steering_wheel
float64 rpm

uint32 lights_high
uint32 lights_medium
uint32 lights_minimum
uint32 lights_left
uint32 lights_right
uint32 lights_brake
uint32 lights_reverse
uint32 lights_warning

uint32 ignition
uint32 emergency

uint32 auto_ignition
uint32 auto_brake
uint32 auto_direction
uint32 auto_clutch